config/environments/development.rb

```diff
@@ -43,4 +40,5 @@
   config.action_mailer.perform_deliveries = false
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.default :charset => "utf-8"
+  ENV['HOST'] = 'hwproj.me'
```

Comment:
Try putting it in `config/application.rb` or `config/environment.rb` to avoid duplication and check which environment it loads in with `Rails.env.production?`.